src/methods/aggregate.jl

```diff
@@ -187,49 +189,55 @@ Disaggregate array, or all arrays in a stack or series, by some scale.
 
 # Keywords
 
-- `progress`: show a progress bar.
+$FILENAME_KEYWORD
+$SUFFIX_KEYWORD
+$PROGRESS_KEYWORD
+$THREADED_KEYWORD
 
 Note: currently it is faster to aggregate over memory-backed arrays.
```

Comment:
should this say disaggregate?

Reply:
It can be deleted, the caching should make it fast now

Reply:
Actually no `cache` is broken on our DiskArrays version 😭